### ISSUE TITLE
Python 3.6 support

### ISSUE THREAD

**Issue by POD666** [olirice/alembic_utils#13]
Looking forward to using alembic_utils in my project but:
```
$ pip install alembic_utils
...
Package 'alembic-utils' requires a different Python: 3.6.9 not in '>=3.7'
```

This issue might be helpful for those who also looking for py36 support so I decided to open it.

Could you add support for py36?

**Comment by olirice** [comment]
Sure, resolved by 80e9afe267177502ad17ee18b5e649de4816d1b3
and available on pypi as `alembic_utils==0.2.9`

**Comment by olirice** [comment]
please reopen if that didn't work for you
